### PR TITLE
Add a contributors section at the bottom of each release note

### PR DESCRIPTION
**Pull request by caithagoras**
We'd like to give credits to all the contributors, similar to what Spark is doing: https://spark.apache.org/releases/spark-release-3-0-0.html

The list of name includes all PR authors and PR committers within the release.

Example Result: https://github.com/prestodb/presto/pull/15291/files#diff-1e8973cca5a81e71ac9c1efdfef41081R45-R48

### CODE REVIEW

File: presto-release-tools/src/test/resources/release-notes-test/release-0.231_expected.rst [hunk 1]
```diff
@@ -21,3 +21,8 @@ ______________
 * Rename error code from ``RAPTOR_LOCAL_FILE_SYSTEM_ERROR`` to ``RAPTOR_FILE_SYSTEM_ERROR``.
 * Remove unused FilterVoidLambda interface in ArrayFilterFunction.
 * Change ``storage.data-directory` from path to URI. For existing deployment on local flash, a scheme header "file://" should be added to the original config value.
+
+**Credits**
```
Comment (ajaygeorge) [inline presto-release-tools/src/test/resources/release-notes-test/release-0.231_expected.rst:25]: `Contributors` reads better. Will leave this to broader community to decide

File: presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java [hunk 2]
```diff
@@ -142,20 +144,26 @@ public void run()
                 .collect(toImmutableMap(identity(), this::extractReleaseNotes));
 
         log.info("Collecting author information");
-        Map<String, String> userByLogin = new HashMap<>();
+        Map<String, String> authorByLogin = new HashMap<>();
```
Comment (ajaygeorge) [inline presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java:147]: Lets add the dedup logic as discussed

**Review by ajaygeorge — APPROVED** [review]
LGTM % few.

+ We need to have a policy to address profanity/hate speech etc in usernames

File: presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java (hunk 2 above)
Comment (caithagoras) [reply presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java]: Used a single login to name map, to dedupe based on login

**Pull request merged**